src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -5,20 +5,111 @@
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int CAPACITY = 16;
```

Comment:
Maybe better to rename it to DEFAULT_CAPACITY for a better understanding of the destination of this variable. Because as for me CAPACITY is a current capacity of the map.

Reply:
done